client/src/components/gamepage/Gamepage.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { } from 'react';
+import PropTypes from 'prop-types';
+import Sidebar from '../sidebar/Sidebar';
+import Roulette from '../roulette/Roulette';
+import ScratchTicket from '../scratch-ticket/ScratchTicket';
+import Slots from '../slots/Slots';
+
+function Gamepage({ user, game }) {
+  const renderGame = () => {
+    if (game === 'roulette') {
+      return <Roulette />;
+    }
+    if (game === 'scratch-ticket') {
+      return <ScratchTicket />;
+    }
+    return <Slots />;
+  };
```

Comment:
this function does not need to be a class method, move it outside the declaration of `Gamepage()`

Reply:
you may also want to use a `switch/case` block here, as the syntax is a little cleaner for this use case